demo/server/src/modules/index.ts

```diff
@@ -94,7 +94,7 @@ function wfmApiSetup(app: express.Router, connectionPromise: Promise<any>) {
   });
 }
 
-function demoDataSetup(connectionPromise: Promise<Db>) {
+function demoDataSetup(connectionPromise: Promise<any>) {
```

Comment:
If the promise that's being passed here is really `Db | undefined` from line 31 I think we should look into where the undefined is coming from, might introduce a bug.

Reply:
That's due to the function that's giving back the connectionPromise as that might be void. https://github.com/feedhenry-raincatcher/raincatcher-core/blob/master/demo/server/src/modules/index.ts#L66-L83

Reply:
Yes, might be worth it to handle that case in this function, even if it's just erroing out and printing it to the user.